doc/source/cd/methods/fetdrift.ipynb

```diff
@@ -0,0 +1,157 @@
+{
```

Comment:
<p>I think we should remove the true/false positive and one-hot examples which exhibit complete dependence rather than independence and therefore applying this detector not really a good idea.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi-detect/pull/359/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Ah yes oops, will remove that right away!